core/modules/replica/AddReplicaQservMgtRequest.cc

```diff
@@ -138,24 +136,42 @@ void AddReplicaQservMgtRequest::startImpl(util::Lock const& lock) {
 
 void AddReplicaQservMgtRequest::finishImpl(util::Lock const& lock) {
 
-    if (extendedState() == ExtendedState::CANCELLED) {
+    switch (extendedState()) {
 
-        // And if the SSI request is still around then tell it to stop
+        case ExtendedState::CANCELLED:
```

Comment:
this is missing a "fallthrough" comment per https://developer.lsst.io/cpp/style.html#a-switch-statement-should-have-the-following-form

Reply:
Technically, this is not a "fallthrough" because there is no single statement separating these cases:
        `case ExtendedState::CANCELLED:`
        `case ExtendedState::TIMEOUT_EXPIRED:`